scripts/actions/fetch-and-deserialize.js

```diff
@@ -69,10 +107,11 @@ const fetchAndDeserialize = (accessToken) => async ({ locale, fileUris }) => {
   );
 
   createDirectories(files);
-
-  files.forEach((file) => writeSync(file, 'utf-8'));
+  writeFilesSync(files);
 };
 
-fetchAndDeserialize();
+if (process.env.CI) {
+  fetchAndDeserialize();
+}
 
-module.exports = fetchAndDeserialize;
+module.exports = { writeFilesSync, fetchAndDeserialize };
```

Comment:
If we change the export signature, I think we'll also need to change how it's imported in other files (`scripts/actions/check-job-prgress.js` comes to mind).

Reply:
oof. good catch.